src/mon/MonClient.cc

```diff
@@ -770,11 +769,26 @@ void MonClient::tick()
 
       send_log();
     }
+
+    if (have_session()) {
```

Comment:
`have_session()` is a method of `MonConnection`. probably we can use the 		1f83eb3 in #16588?